### PR TITLE
Add Adobe Photoshop Lightroom 4.4.1

### PR DESCRIPTION
**Pull request by elliotthilaire**
Lightroom 4 previously existed as lightroom4.rb. It was removed here 9abfcf267b78a8a5adbc04a0eb1fccacdfaab002

Re-adding it and updating to 4.4.1 in a manner that is consistent with adobe-photoshop-lightroom571.rb

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/adobe-photoshop-lightroom441.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'adobe-photoshop-lightroom441' do
+  version '4.4.1'
+  sha256 '6d703aa5ea0022141c88746829e80c60f1169d746022655937c4196a6ca35dad'
+
+  url "http://download.adobe.com/pub/adobe/lightroom/mac/#{version.to_i}.x/Lightroom_#{version.to_i}_LS11_mac_#{version.gsub('.', '_')}.dmg"
```
Comment (vitorgalvao) [inline Casks/adobe-photoshop-lightroom441.rb]: Use `.major` instead pf `.to_i` and `.dots_to_underscores` instead of `.gsub('.', '_')`.

Comment (elliotthilaire) [inline Casks/adobe-photoshop-lightroom441.rb]: No problem. Want me to update the Lightroom 5 cask with `.major` and `.dots_to_underscores` while I am at it?

Comment (fanquake) [inline Casks/adobe-photoshop-lightroom441.rb]: @elliotthilaire sure, thanks.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**